src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexWriter.cs

```diff
@@ -299,7 +262,7 @@ private RegexCode RegexCodeFromRegexTree(RegexTree tree)
 
             anchors = RegexFCD.Anchors(tree);
 
-            return new RegexCode(_emitted, _stringtable, _trackcount, _caps, capsize, bmPrefix, fcPrefix, anchors, rtl);
+            return new RegexCode(_emitted.ToArray(), _stringtable, _trackcount, _caps, capsize, bmPrefix, fcPrefix, anchors, rtl);
```

Comment:
ToArray copies the elements into a new array. It might make more sense to just return the List<int> as an IEnumerable<int> here.

Reply:
As an IEnumerable?  Doesn't RegexCode need to be able to index into it?  If anything I'd expect you'd just store the existing List as a List.  But as Dan said earlier, everything should be supported by measurements.